dbsync.py

```diff
@@ -34,6 +34,12 @@ class DbSyncError(Exception):
     pass
 
 
+def _add_quotes_to_schema_name(schema: str) -> str:
+    if any(ele.isupper() for ele in schema):
+        schema = f'"{schema}"'
+    return schema
```

Comment:
this escaping should be probably more elaborate - e.g. what if schema name contains space... or a double quotes :-) - hence https://www.psycopg.org/docs/sql.html#psycopg2.sql.Identifier may be a better choice than our own implementation

Reply:
thanks. Schema names are independent from MM project name, so it is good to have the escaping there...